cmake/tribits/core/package_arch/TribitsGlobalMacros.cmake

```diff
@@ -254,12 +254,9 @@ MACRO(TRIBITS_DEFINE_GLOBAL_OPTIONS_AND_DEFINE_EXTRA_REPOS)
     "Enable the C++ compiler and related code"
     ${${PROJECT_NAME}_ENABLE_CXX_DEFAULT} )
 
-  IF ("${${PROJECT_NAME}_ENABLE_CXX11_DEFAULT}" STREQUAL "")
-    SET(${PROJECT_NAME}_ENABLE_CXX11_DEFAULT OFF)
-  ENDIF()
-  ADVANCED_OPTION(${PROJECT_NAME}_ENABLE_CXX11
-    "Enable the C++11 compiler options and related code (see ${PROJECT_NAME}_CXX11_FLAGS)"
-    ${${PROJECT_NAME}_ENABLE_CXX11_DEFAULT} )
+  # Hard-code a variable with the same name as a now-removed option that is always enabled.
+  # This can be removed after clients have been updated.
+  SET(${PROJECT_NAME}_ENABLE_CXX11 ON)
```

Comment:
I wonder if this is really needed.  With the Trilinos 13.0 release coming up, it might be better to just remove this.  This might be confusing when Trilinos requires C++14 (see #6260) to have a variable `Trilinos_ENABLE_CXX11` set to `ON`.

Reply:
On second thought, it is better to keep this in for now.  I am just wondering how we can get rid of this variable later in client code so they will not be confused when Trilinos requires C++14 support.